css/grayscale.scss

```diff
@@ -351,6 +351,19 @@ a{
     color: white;
 }
 
+@media(min-width:768px) {
```

Comment:
is this change related to contacts?

Reply:
may be we need this to be enlarged in web view.

Reply:
Exactly what it is done there if the screen width is 786px or more it will enlarge.

Reply:
got it. Thanks.